docs/finetuning.md

```diff
@@ -10,6 +10,44 @@ model = AuroraPretrained()
 model.load_checkpoint()
 ```
 
+## Basic Fine-Tuning Environment
+
+We provide a very basic Docker image and fine-tuning loop to get you started.
+The Docker image can be found at `finetuning/Dockerfile` and the fine-tuning
+loop at `finetuning/finetune.py`.
+You can build and run the image as follows:
+
+```bash
+docker build . -t aurora:latest -f finetuning/Dockerfile
+docker run --rm -it -v .:/app/aurora \
```

Comment:
An issue may arise here if the user already has a `.venv` folder locally as that would get mounted. I think to solve this, you need to put an anonymous mount like in https://docs.astral.sh/uv/guides/integration/docker/#mounting-the-project-with-docker-run

In particular, you might need to do `-v /app/.venv`